### ISSUE TITLE
k0s default-config sans has same ips twice

### ISSUE THREAD

**Issue by matti** [k0sproject/k0s#520]
is this intentional?

on scaleway instance:

```
$ k0s default-config
apiVersion: k0s.k0sproject.io/v1beta1
kind: Cluster
metadata:
  name: k0s
spec:
  api:
    address: 10.18.80.227
    sans:
    - 10.18.80.227
    - 10.18.80.227
```

^-- two same sans?

```
$ ip address
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: ens2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether de:2e:40:06:50:72 brd ff:ff:ff:ff:ff:ff
    inet 10.18.80.227/31 brd 10.18.80.227 scope global ens2
       valid_lft forever preferred_lft forever
    inet6 2001:bc8:1824:71::1/64 scope global
       valid_lft forever preferred_lft forever
    inet6 fe80::dc2e:40ff:fe06:5072/64 scope link
       valid_lft forever preferred_lft forever
```

**Comment by jnummelin** [comment]
Not really intentional, but should be pretty harmless.

I think it comes from this line: https://github.com/k0sproject/k0s/blob/main/pkg/apis/v1beta1/cluster.go#L160

**Comment by jnummelin** [comment]
fixed in #663
